File: 20260324_Contenido/20260324_Contenido_1_info.py
"""
Video: youtube.com/shorts/1n5HHIuKN34

Cómo remover elementos duplicados de una lista con Python:

Podemos utilizar la función set() para crear un conjunto y luego revertirlo, el cual remueve los elementos duplicados. Después imprimimos el resultado.
"""

lista1 = [1, 2, 2, 3, 4, 3, 5]
print(list(set(lista1)))

File: 20260324_Contenido/20260324_Contenido_1_info.js
/*
Video: youtube.com/shorts/NJRnFrysdMw

Cómo remover elementos duplicados de una matriz con JavaScript:

Podemos convertir la matriz en un objeto Set y luego revertirlo, el cual remueve los elementos duplicados. Después mandamos el resultado a la consola.
*/

let matriz1 = [1, 2, 2, 3, 4, 3, 5];
console.log(Array.from(new Set(matriz1)));
